### ISSUE TITLE
DoT on Cloudflare Workers

### ISSUE THREAD

**Issue by luxysiv** [serverless-dns/serverless-dns#65]
can't connect when add dns over tls on setting Android 11
![Screenshot_20220326-090629~2](https://user-images.githubusercontent.com/46205571/160220761-c42d53f0-dcc1-4f9b-8ecb-c4d82c69fc64.png)
please fix that ( wokers cloudflare)

**Comment by ignoramous** [comment]
DNS over TLS is only supported for deploys on Fly.io right now ([steps](https://docs.rethinkdns.com/dns/open-source#fly-io)).

We do intend to support DoT (private dns) once Workers supports raw TCP workloads: [Workers TCP](https://blog.cloudflare.com/introducing-socket-workers/)

For now, you'd have to use a DoH client like [Nebulo](https://github.com/ch4t4r/Nebulo) or [RethinkDNS+Firewall](https://github.com/celzero/rethink-app) to connect to your Workers endpoint.

**Comment by Kaito273** [comment]
> DNS over TLS is only supported for deploys on Fly.io right now ([steps](https://docs.rethinkdns.com/dns/open-source#fly-io)).
> 
> We do intend to support DoT (private dns) once [Workers supports raw TCP workloads](https://blog.cloudflare.com/introducing-socket-workers/).
> 
> For now, you'd have to use a DoH client like [Nebulo](https://github.com/ch4t4r/Nebulo) or [RethinkDNS+Firewall](https://github.com/celzero/rethink-app) to connect to your Workers endpoint.

It has been a year. Can you please share any new developments regarding this issue? Did workers started supporting raw TCP workloads?

**Comment by ignoramous** [comment]
Nope. Cloudflare only supports outgoing TCP connections, right now.
